packages/cli-lib/README.md

```diff
@@ -0,0 +1,71 @@
+## What is CLI-lib for?
+
+HubSpot's cli-lib aims to provide useful ways of interacting with HubSpot's public APIs.
+
+## Framework-agnostic Integrations
+
+HubSpot's cli-lib can integrate with various task runners and bundlers such as Gulp, Grunt, and Webpack.
+
+Internally, the HubSpot cli-lib is consumed by the HubSpot CLI, the official HubSpot Webpack plugin, and the official HubSpot serverless development runtime.
+
+## Functionality
+
+The full scope of the cli-lib is quite large. It provides the ability to manage authentication, custom objects, the file manager, files within the Design Manager, serverless functions, HubDB, and more.
```

Comment:
I appreciate the idea of starting high-level and exploring the use cases, but I wonder if Instead of leaving the functionality open ended could we be more thorough in describing the different features and perhaps go as far as outlining the public interface (e.g. the modules + functions) so that we can be clearer when it comes to what the key pieces are for developers.

Reply:
Yeah that's the next step.  This was just the initial overview step

Reply:
Before documenting the public interface, though, we need to figure out what we want in the public interface